libs/chain-signatures/contract/tests/updates.rs

```diff
@@ -31,7 +31,7 @@ pub fn invalid_contract() -> ProposeUpdateArgs {
 /// This is the current deposit required for a contract deploy. This is subject to change but make
 /// sure that it's not larger than 2mb. We can go up to 4mb technically but our contract should
 /// not be getting that big.
-const CURRENT_CONTRACT_DEPLOY_DEPOSIT: NearToken = NearToken::from_millinear(9820);
+const CURRENT_CONTRACT_DEPLOY_DEPOSIT: NearToken = NearToken::from_millinear(13000);
```

Comment:
JFYI: IIUC, I’ll likely be able to reduce that value again once I integrate the attestation module with the contract, since I’ll need to remove a lot of the existing code in the contract.